xlators/mgmt/glusterd/src/glusterd-volume-set.c

```diff
@@ -3103,4 +3103,9 @@ struct volopt_map_entry glusterd_volopt_map[] = {
      .op_version = GD_OP_VERSION_9_0,
      .value = "yes",
      .flags = VOLOPT_FLAG_CLIENT_OPT},
+
+    {.key = "distribute.ensure-durability",
```

Comment:
should we call this option `rebalance.ensure-durability` for better relation to the actual feature ?